corehq/apps/es/case_search.py

```diff
@@ -221,6 +221,23 @@ def case_property_text_query(case_property_name, value, operator=None):
     )
 
 
+def case_property_starts_with_query(case_property_name, value):
+    """Filter by case_properties.key and do a text search in case_properties.value that
+       matches starting substring.
+
+    """
+    return queries.nested(
+        CASE_PROPERTIES_PATH,
+        queries.filtered(
+            queries.match_all(),
+            filters.AND(
+                filters.term(PROPERTY_KEY, case_property_name),
+                filters.prefix(PROPERTY_VALUE_EXACT, value),
+            )
+        )
```

Comment:
I believe you can drop the `queries.filtered` wrapper here
```suggestion
        filters.AND(
            filters.term(PROPERTY_KEY, case_property_name),
            filters.prefix(PROPERTY_VALUE_EXACT, value),
        )
```

Reply:
Incorporated in commit https://github.com/dimagi/commcare-hq/pull/32119/commits/005f39dcd060b743ab5c762c0a2211e077859799